docs/examples/rx/05_read_storage/index.js

```diff
@@ -0,0 +1,55 @@
+import { combineLatest } from 'rxjs';
+import { switchMap, map } from 'rxjs/operators';
+
+// Import the API
+const { ApiRx } = require('@polkadot/api');
+
+// Our address for Alice on the dev chain
+const Alice = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+
+async function main () {
+  // Create the API and wait until ready using default provider.
+  // Subscribe to the basic chain state/storage queries
+  // Use the Storage chain state (runtime) Node Interface.
+  const subscriptionApiRx = ApiRx.create()
```

Comment:
This example is completely un-followable.

- Api.Rx.create().toPromise()
- validators as seperate function

The idea is not to be overly verbose, but at the same time you shouldn't need a PhD in optimisation to be able to follow the code. As examples things should be clear and understandable.